crates/client/rpc-core/src/lib.rs

```diff
@@ -27,9 +27,26 @@ use starknet_core::types::{
 #[derive(Serialize, Deserialize)]
 pub struct Felt(#[serde_as(as = "UfeHex")] pub FieldElement);
 
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccount {
+    pub contract_address: FieldElement,
+    pub contract_class: ContractClass,
```

Comment:
Is there any specific reason to return the `contract_class`? I am not sure how it will be used by the client and it make the response really big.

Reply:
You can return the ContractClassHash. It's informative and light

Reply:
No real reason, I wasn't aware the response would be that big until I tested it. I can definitely replace this attribute with a class hash.